modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -701,6 +704,8 @@ RenderablePointCloud::RenderablePointCloud(const ghoul::Dictionary& dictionary)
 
     _useCaching = p.useCaching.value_or(true);
 
+    _skipFirstDataPoint = p.skipFirstDataPoint.value_or(false);
```

Comment:
```suggestion
    _skipFirstDataPoint = p.skipFirstDataPoint.value_or(_skipFirstDataPoint);
```

That way, if we ever decide to change the default value in the header, we don't have the chance to miss that change here